modules/mod_ginger_collection/lib/js/beeldenzoeker.js

```diff
@@ -56,13 +56,21 @@ $(document).ready(function(){
 
 
 function homeInfinite() {
+	staticWindowHeight = $(window).height();
```

Comment:
Spaces, not tabs, as described in our [.editorconfig](https://github.com/driebit/ginger/blob/master/.editorconfig). Please install an Editorconfig plugin for your IDE.